### DISCUSSION TITLE
请问能增加一个动态页面里分组浏览吗

### DISCUSSION THREAD

**Discussion by eno1016** [the1812/Bilibili-Evolved#1586] [功能建议 / Ideas]
动态页面里，只浏览某个分组，或者只浏览某几个分组，可以实现吗大佬。。。
关注了个b站纪录片，每天都在动态里面被刷屏，偶尔想看看影视吐槽或者游戏分组几个大佬的动态，完全找不到在哪。。。
油猴倒是有个脚本可以实现，但是不知道是不是我电脑的问题，用起来有点卡卡的。。。而且没办法保存设置，只要一刷新就要重新再设置一遍分组动态。
非常需要这个功能，不知道大佬有没有兴趣能实现一下这个。。
感谢大佬！

**Comment by Geeyun-JY3** [comment]
之前有人提过：#1279
建议你先去跟这个脚本作者反馈
